### ISSUE TITLE
Doesn't work with Aurora

### ISSUE THREAD

**Issue by narrill** [Brusalk/EventHorizon_Continued#110]
Event Horizon doesn't work with [this](https://wow.curseforge.com/projects/aurora) addon installed and enabled. It gets as far as parsing the current class config, but doesn't get to a state where the frame can be shown or the /ehz ... commands can be used.

**Comment by narrill** [comment]
I figured out the problem. EventHorizon tries to get class colors from `CUSTOM_CLASS_COLORS` before defaulting to `RAID_CLASS_COLORS`, but Aurora initializes `CUSTOM_CLASS_COLORS` to what I assume is a non-standard format, causing EventHorizon to crash.

I don't know what `CUSTOM_CLASS_COLORS` is supposed to be used for (e.g. whether convention allows it to be sparse), so I don't know whether this is Aurora doing something it shouldn't or EventHorizon not validating things properly. I've opened an issue with Aurora as well, so we'll see what happens; I have a fix I can use locally in the mean time at least.
